tests/core/test_subdir_data.py

```diff
@@ -307,12 +310,47 @@ def test_metadata_cache_clearing(platform=OVERRIDE_PLATFORM):
         assert precs_b == precs_a
 
 
-# @pytest.mark.integration
-# class SubdirDataTests(TestCase):
-#
-#     def test_basic_subdir_data(self):
-#         channel = Channel("https://conda.anaconda.org/conda-test/linux-64")
-#         sd = SubdirData(channel)
-#         sd.load()
-#         print(sd._names_index.keys())
-#         assert 0
+def test_cache_json(tmp_path: Path):
+    """
+    Load and save standardized field names, from internal matches-legacy
+    underscore-prefixed field names. Assert state is only loaded if it matches
+    cached json.
+    """
+    cache_json = tmp_path / "cached.json"
+    cache_state = tmp_path / "cached.state.json"
+
+    cache_json.write_text("{}")
+
+    CacheJsonState(cache_json, cache_state, "repodata.json").save()
+
```

Comment:
Could we check for the returned path here?

Reply:
`write_text` returns the number of bytes written, not a Path